tornado-drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/graal/nodes/OCLFPTernaryIntrinsicNode.java

```diff
@@ -100,6 +81,34 @@ protected static ValueNode tryConstantFold(ValueNode x, ValueNode y, ValueNode z
         return result;
     }
 
+    private static double doCompute(double x, double y, double z, Operation op) {
+        switch (op) {
+            default:
+                throw new TornadoInternalError("unable to compute op %s", op);
+        }
+    }
+
+    private static float doCompute(float x, float y, float z, Operation op) {
+        switch (op) {
+            default:
```

Comment:
remove the swtich statement

Reply:
I didn't add this code, it was just moved around due to auto-formatting

Reply:
I fixed it